### ISSUE TITLE
Missing 'album' Field in Recommendations.TrackSimple from recommendations.get API

### ISSUE THREAD

**Issue by tikbox** [rinukkusu/spotify-dart#176]
Issue Summary:
The recommendations.get API endpoint is returning incomplete data for the Recommendations.TrackSimple object. Specifically, the 'album' field is missing from the response, despite it being documented in the official API documentation. This issue is impacting the ability to retrieve complete information about recommended tracks.

Expected Behavior:
The Recommendations.TrackSimple object should include the 'album' field in the response, as specified in the official API documentation.

Thank you for your attention to this matter.

**Comment by rinukkusu** [comment]
You are right - looking at the [documentation](https://developer.spotify.com/documentation/web-api/reference/get-recommendations) there is indeed an `album` field in the example response. On the other hand the `tracks` array itself is made up of simplified versions of the `Track` object, as stated in that same documentation, which should not contain an album.

> An array of track object (simplified) ordered according to the parameters supplied.

I think this is something we could tackle with the model classes rewrite in issue #172 - so we could have like a mixin or something for this specific case.

For now we could switch the `tracks` array to an array of the "full" `Track` class. Some fields might be empty of course. What do you think @hayribakici ?

**Comment by hayribakici** [comment]
Makes sense, @rinukkusu.
